packages/router/src/grpc/view-protocol-server/balances.ts

```diff
@@ -109,6 +120,39 @@ class BalancesAggregator {
     );
     currentAmount.lo = newAmount.lo;
     currentAmount.hi = newAmount.hi;
+
+    await this.aggregateEquivalentValues(valueView, toAdd);
+  }
+
+  private async aggregateEquivalentValues(valueView: ValueView, toAdd: SpendableNoteRecord) {
```

Comment:
Can we get a brief doc string about how this is aggregating?

Reply:
done